lang/java/pom.xml

```diff
@@ -35,39 +35,40 @@
 
   <properties>
     <main.basedir>${project.parent.basedir}</main.basedir>
+    <java.version>1.8</java.version>
```

Comment:
I think `java.version` is a predefined system property.
You'll need to use a different name here.